### PR TITLE
chore(Storybook): refactor all cards

### PR DESCRIPTION
**Pull request by marcoskolodny**
Continuation of the storybook refactor: [Issue](https://jira.tid.es/browse/WEB-819)

### CODE REVIEW

File: src/__stories__/data-card-story.tsx [hunk 1]
```diff
@@ -164,7 +164,7 @@ export const Group: StoryComponent = () => {
             <Stack space={16}>
                 <Text2 regular>
                     We can group multiple cards and they adjust to the same height. The card actions are
-                    always fixed on bottom:
+                    always fixed on bottom
                 </Text2>
                 <style>{`.group > * {width: 300px}`}</style>
                 <Inline space={16} className="group">
```
Comment (atabel) [inline src/__stories__/data-card-story.tsx:170]: as discussed in Teams, please migrate group stories to use Carousel instead of custom styles

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/__stories__/display-media-card-story.tsx [hunk 2]
```diff
@@ -107,84 +110,35 @@ export const Default: StoryComponent<DisplayMediaCardArgs> = ({
                             },
                         ]
                       : undefined,
-                  backgroundImage: BACKGROUND_IMAGE_SRC,
+                  backgroundImage: isEmptySource ? '' : BACKGROUND_IMAGE_SRC,
```
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/display-media-card-story.tsx:113]: In image/video story, we have a boolean control for isEmptySource. I think it's more consistent to follow the same approach

File: src/__stories__/snap-card-story.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ import avatarImg from './images/avatar.jpg';
 import type {AspectRatio} from '../card';
 
 export default {
-    title: 'Components/Cards/Snap card',
+    title: 'Components/Cards/SnapCard',
 };
 
 type Args = {
```
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/snap-card-story.tsx:23]: Fixed. Now all cards have "actions" story control with onPress/none/button/etc

File: src/__stories__/media-card-story.tsx
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/media-card-story.tsx:1]: Good call. Fixed in media and naked card

File: src/__stories__/data-card-story.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/__stories__/data-card-story.tsx]: updated all card group stories to use carousel

File: src/__stories__/videos/confetti.mp4
Comment (marcoskolodny) [inline src/__stories__/videos/confetti.mp4:1]: Unrelated change: @yceballost requested me to remove this video from all stories and use the beach one instead

File: src/highlighted-card.tsx [hunk 4]
```diff
@@ -76,6 +76,7 @@ const Content = React.forwardRef<HTMLDivElement, Props>((props, ref) => {
             className={styles.container}
             dataAttributes={{'component-name': 'HighlightedCard', ...props.dataAttributes}}
             width={props.width ? `${props.width}px` : '100%'}
+            minHeight="100%"
```
Comment (marcoskolodny) [inline src/highlighted-card.tsx:79]: Highlighted card was not adapting it's height to the container (like all other cards do).

**Review by atabel — APPROVED** [review]

File: src/__stories__/videos/confetti.mp4
Comment (yceballost) [reply src/__stories__/videos/confetti.mp4:1]: kill the confetti video 🔪

**Review by yceballost — APPROVED** [review]

**Review by pladaria — APPROVED** [review]
